enhancedTable/index.ts

```diff
@@ -1,6 +1,7 @@
 import { GridOptions, GridApi } from 'ag-grid-community';
 import { take } from 'rxjs/internal/operators/take';
 import { PanelManager } from './panel-manager';
+import { ComponentResolver } from 'node_modules/ag-grid/dist/lib/components/framework/componentResolver';
```

Comment:
What does this do?

Reply:
Not sure. I think it appeared after the rebase. Removed